### PR TITLE
Remove deprecated cpfs plugin

### PR DESCRIPTION
**Pull request by tydra-wang**
#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
The old cpfs plugin is deprecated, users should turn to the nas plugin for mounting CPFS(2.0) protocol server.
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by huww98** [comment]
Please also remove

./docs/cpfs-dynamic.md
./docs/cpfs.md
./examples/cpfs
./build/cpfs
./build/build-cpfs.sh

File: main.go [hunk 1]
```diff
@@ -192,11 +191,7 @@ func main() {
 			}(endPointName)
 
 		case TypePluginCPFS:
-			go func(endPoint string) {
-				defer wg.Done()
-				driver := cpfs.NewDriver(*nodeID, endPoint)
-				driver.Run()
-			}(endPointName)
+			log.Fatalf("%s is already deprecated, please switch to %s if you are using CPFS 2.0 protocol server", TypePluginCPFS, TypePluginNAS)
```
Comment (huww98) [inline main.go:194]: "deprecated" usually means not recommended but still usable. Maybe use word "removed"/"dropped"/"unsupported"?

Comment (mowangdk) [reply main.go:194]: "unsupported" sounds good

**Pull request merged**